source/platforms/databricks.md

```diff
@@ -66,6 +88,24 @@ gdf
 
 ```
 
+Connect to the dask client and submit tasks.
+
+```python
+from dask.distributed import Client
+from dask_databricks import DatabricksCluster
+
+cluster = DatabricksCluster()
+client = Client(cluster)
+
+def inc(x):
+    return x + 1
+
+x = client.submit(inc, 10)
+x.result()
+ 11
```

Comment:
It would be good to have a cudf example here. Maybe something like

```python
import cudf
import dask

df = dask.datasets.timeseries().map_partitions(cudf.from_pandas)
print(df.x.mean().compute())
```

Or even something better!